build/modify-site.mjs

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
+
+/*!
+ * Script to modify relative paths in generated OUDS Web documentation.
+ *
+ * Copyright (c) 2015-2024 Orange SA
+ * Copyright (c) 2015-2024 The OUDS Web Authors
+ * Licensed under MIT (https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/ouds/main/LICENSE)
+ */
+
+import sh from 'shelljs'
+
+if (sh.test('-d', 'ouds-web')) {
+  sh.rm('-rf', 'ouds-web')
+}
+
+sh.mkdir('ouds-web')
+sh.mv('_site/*', 'ouds-web/')
+sh.mv('ouds-web', '_site/')
+
+for (const file of sh.find('_site').filter(filename => {
+  return filename.includes('.html') || filename.includes('.xml')
```

Comment:
Why do we need this filter ?

Reply:
Because you can't `sed` a directory. The syntax was easier than checking that it's a file. At least the ones I've found.